src/theme/Navbar/index.js

```diff
@@ -14,11 +14,10 @@ import isInternalUrl from '@docusaurus/isInternalUrl';
 import { useLocation } from '@docusaurus/router';
 
 import SearchBar from '@theme/SearchBar';
-import Toggle from '@theme/Toggle';
+// import Toggle from '@theme/Toggle';
```

Comment:
Why was this code commented out? Just delete it if that's your solution. I'll push the fix.